docs/UserGuide.md

```diff
@@ -3,44 +3,122 @@ layout: page
 
-# Quick start
+## 4. Using This Guide
 
-1. Ensure you have Java `11` or above installed in your Computer.
+### 4.1. Navigate Around
 
-2. Download the latest `SoConnect.jar` from [here](https://github.com/AY2223S1-CS2103T-W15-1/tp/releases).
+As a first time user, we understand that you might be confused on how to even start up the application in the first place. Therefore, you can go to the [Installation](#5-installation) section to download SoConnect, if you have not done so.
 
-3. Copy the file to the folder you want to use as the _home folder_ for your SoConnect.
+Once SoConnect is installed, you can navigate to the [Quickstart](#6-quickstart) section, where we will cover the basics of using SoConnect.
 
-4. Double-click the file to start the app. A GUI similar to the one below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+If you are a returning user, you can head over to the [Command Summary](#command-summary) section for a quick overview of all SoConnect's commands. Alternatively, you can also read the detailed explanation of each command in the [Commands](#commands) section.
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
-   Some example commands you can try:
+Feeling stuck? The [FAQ](#faq) section might help find the answer you are looking for.
 
-   * **`list`** : Lists all contacts.
+You can also refer to the [Glossary](#glossary) section for definitions of commonly used terms in SoConnect.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to your SoConnect.
+### 4.2. Icons
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+In order to help you pay attention, we have used some icons along with highlighted text throughout this guide.
 
-   * **`clear`** : Deletes all contacts.
+| Icon                          | Meaning                                       |
+|-------------------------------|-----------------------------------------------|
+| **:memo: Note**               | You should pay attention to this.             |
+| **:bulb: Tip**                | You might find this useful.                   |
+| **:information_source: Info** | You can get additional information from this. |
+| **:warning: Warning**         | You should be cautious about this.            |
+| **:rotating_light: Danger**   | You should pay **extra** attention to this.   |
 
-   * **`exit`** : Exits SoConnect.
+### 4.3. Formatting and Notation
 
-6. Refer to the [Features](#features) below for details of each command.
+In case you are wondering why certain texts are styled differently, here are the meaning behind:
+
+1. Words in big size, numbered and in purple color are section titles and subtitles.
+2. **Words in bold** are phrases that we want you to read.
```

Comment:
I am not sure about the important part. Because if you notice, the bold is actually used to catch the user's attention, more like a promotional tool, I would say.